docs/embedding/sdk/dashboards.md

```diff
@@ -159,14 +159,14 @@ You can add custom actions to the dashcard menu by adding an object to the `cust
     iconName: string;
     label: string;
     onClick: () => void;
-    disabled ? : boolean;
+    disabled?: boolean;
 }
 ```
 
 Here's an example:
```

Comment:
The `:` is still there, the diffs is a bit misleading

Reply:
This color probably means it's an invalid syntax as far as I can tell.

Reply:
@WiNloSt the rich diff preview shows up fine though, so that confuses me a bit

<img src="https://github.com/user-attachments/assets/fc0eff5a-24f1-496a-9357-9fcd2f2b2d01" width="400">

Reply:
yeah, GH can't always be right can it 😄